cli/src/account.rs

```diff
@@ -41,7 +41,8 @@ pub async fn account_prompt(account: Account) -> Result<(), Error> {
 }
 
 // loop on the account prompt
-pub async fn account_prompt_internal(account: Account, history: &mut AccountHistory) -> Result<bool, Error> {
+pub async fn account_prompt_internal(wallet: Wallet, account: String, history: &mut AccountHistory) -> Result<bool, Error> {
```

Comment:
Why take ownership of the Wallet ?

Reply:
Forces you to clone for no reason

Reply:
you're right ... kind of expected we would need it since we claimed ownership for `Account` before this PR and that wasn't necessary either.